contentrepo/settings/dev.py

```diff
@@ -16,10 +16,6 @@
 FB_BUSINESS_ID = "27121231234"
 
 
-# NOTE: We don't want the cache getting in the way during tests, but this also
-# means we're not testing the cache behaviour.
-CACHES = {
-    "default": {
-        "BACKEND": "django.core.cache.backends.dummy.DummyCache",
-    }
-}
+# We default to using local memory for dev and tests, so that Redis isn't a dependancy
+# We test against Redis in the CI
+CACHES = {"default": env.cache("CACHE_URL", default="locmemcache://")}
```

Comment:
Do we really want in-memory caching in tests?

Reply:
I thought it might be useful, as that's how we originally picked up that the cache wasn't being invalidated on changes, by a test failing. Now that those endpoints are no longer cached, that issue is fixed, so I thought we could turn it back on again. If we do end up adding caching back again, then we'll want it on, so that we can test that our caching is working correctly.

Happy either way though. I've got the CI testing against redis, to better simulate the production environment, but if we want it off for tests we must just remove redis there as well.

Reply:
Ah, right. The internal caching (which is mostly ORM-level, I guess?) is safe -- it's just the HTTP response caching that bit us. Happy with this.